drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -117,6 +117,11 @@ matterGeneric:
     deviceTypes:
       - id: 0x002C
     deviceProfileName: air-quality-sensor
+  - id: "matter/air-quality-ikea"
+    deviceLabel: IKEA air Quality Sensor
+    deviceTypes:
+      - id: 0x002C # Air Quality
+    deviceProfileName: air-quality-ikea
   - id: "matter/air-quality-tuo"
```

Comment:
@ctowns This isn't added in this PR, but I'm not sure we should have this device specific profile name for a device with just generic device type fingerprints.

Reply:
This fingerprint was removed as recommended in another comment.